src/utils/cf/index.ts

```diff
@@ -28,7 +28,15 @@ export function hasAxis(
 }
 
 function isInterval(maybeInterval: string): maybeInterval is ManipulateType {
-  return ["milliseconds", "seconds", "minutes", "hours", "days"].includes(
+  return [
+    "nanoseconds",
+    "microseconds",
```

Comment:
Did you verify this is working?
Accoding to the [dayjs docs](https://day.js.org/docs/en/manipulate/add#list-of-all-available-units), "nanoseconds" and "microseconds" are no acceptable units. We thus probably have to do some special handling (e.g. rescaling) before passing the values to dayjs or add those units to dayjs (I don't know how atm.)

Reply:
> I see that nanoseconds might cause overflows when used with non-Python libraries (e.g. JavaScript). However, the position/attitude dataset currently always encodes as nanoseconds (even if set explicitly)

It causes an overflow when printing the time range. I can try to massage the position/attitude datasets a bit more

Reply:
As long as calculations stay in [BigInt](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/BigInt) types or [Int64 TypedArrays](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/TypedArray), there shouldn't be an overflow. But as soon as you convert them to [Number](https://developer.mozilla.org/de/docs/Glossary/Number), which is essentially a 64bit Float, there might be overflow / underflow issues. But this should be the same for JavaScript and Python in the same ways.

Reply:
Then the overflow likely occurs in my dirty hack in the minmax function.

But I can also try to [force non-nanosecond times](https://github.com/pydata/xarray/discussions/6712#discussioncomment-2997781) into the Xarray dataset

Reply:
I tried to implement something and on first sight, it seems to work.